integration-testing/test.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+set -Euxo pipefail
+
+if ! lein with-profile -user,-dev install; then
+  exit 1
+fi
+
+cd integration-testing/red1 || exit 1
+
+if lein with-profile -user antq; then
+  echo "Should have failed!"
+  exit 1
+fi
+
+cd ../red2 || exit 1
+
+if lein with-profile -user antq; then
+  echo "Should have failed!"
+  exit 1
+fi
+
+cd ../red3 || exit 1
+
+if lein with-profile -user antq; then
+  echo "Should have failed!"
+  exit 1
+fi
+
+cd ../green || exit 1
+
+if ! lein with-profile -user antq; then
+  echo "Should have passed!"
+  exit 1
+fi
+
+exit 0
```

Comment:
I want to move this file to `script/integration-test.sh` because I'd like to keep all script files here.